var/spack/repos/builtin/packages/brainbuilder/package.py

```diff
@@ -16,6 +16,7 @@ class Brainbuilder(PythonPackage):
     version('0.16.1', tag='brainbuilder-v0.16.1')
 
     variant('reindex', default=True, description='Install requirements for reindex')
+    variant('subcellular', default=True, description='Install requirements for subcellular')
```

Comment:
Do you ever use/disable these variants? Otherwise we could just skip all that

Reply:
Do you mean that we could include all the dependencies in the default version?
I added them separately because they are separate features defined in `extras_require` in `setup.py`.
However, I've set `default=True` because it should be simpler for the users needing them.
Unless anyone has reasons to keep them, I can remove the variants and keep the dependencies.
The result for the user will be the same (but he won't be able to load the module without the optional features).

Reply:
Yes, I'd sort of include all dependencies (maybe make a note which pip feature they correspond to). I don't think we have any way to control the installed extras, so variants are sort-of cosmetic IMO.